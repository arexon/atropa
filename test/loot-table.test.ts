import { describe, expect, it } from 'vitest'
import lootTable from '#playground/BP/loot_tables/radio.loot'

describe('define loot table', () => {
	it('returns a valid loot table object', () => {
		expect(lootTable).toMatchSnapshot()
	})
})
