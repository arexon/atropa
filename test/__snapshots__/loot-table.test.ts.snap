// Vitest Snapshot v1

exports[`define loot table > returns a valid loot table object 1`] = `
{
  "pools": [
    {
      "entries": {
        "name": "test:table",
        "type": "item",
        "weight": 1,
      },
      "rolls": 1,
    },
  ],
}
`;
